src/content/app/entity-viewer/state/gene-view/transcripts/geneViewTranscriptsSlice.ts

```diff
@@ -171,6 +173,14 @@ export const toggleTranscriptInfo =
         expandedIds: [...expandedIds.values()]
       })
     );
+
+    dispatch(
+      transcriptsSlice.actions.updateAccordionUserInteraction({
+        activeGenomeId,
+        activeEntityId,
+        accordionUserInteraction: true
+      })
+    );
```

Comment:
It doesn't look like we need this action at all. The boolean flag field can be modified directly in the `updateExpandedTranscripts` reducer.